README.md

```diff
@@ -8,43 +8,38 @@
 
 GitHub CLI is available for repositories hosted on GitHub.com and GitHub Enterprise Server 2.20+, and to install on macOS, Windows, and Linux. 
 
-
 ## Documentation
 
 Read the [official docs][] for usage and more information.
 
-
-
 ## We want your feedback
 
 We'd love to hear your feedback about `gh`. If you spot bugs or have features that you'd really like to see in `gh`, please check out the [contributing page][].
 
-
-
 <!-- this anchor is linked to from elsewhere, so avoid renaming it -->
 ## Installation
 
 ### macOS
 
-`gh` is available via [Homebrew][] and [MacPorts][].
+`gh` is available via [Homebrew][] and [MacPorts][], and a downloadable binary from the [releases page][].
```

Comment:
I think this reads better as `via [Homebrew][], [MacPorts][], and as a downloadable binary`

Reply:
Are you suggesting moving the link to "downloadable binary", or removing it altogether?

Reply:
Ahh sorry, my comment was not very clear. I just meant replacing the `and` between Homebrew and Macports with a `,`. Then adding an `as` before "a downloadable binary". I was trying to save typing by not writing out the whole line. 

This is what I think this line should look like:
```
`gh` is available via [Homebrew][], [MacPorts][], and as a downloadable binary from the [releases page][].
```

Reply:
Oh! Right, I totally missed that the sentence would have two `and`s when I first edited it. I've now edited the line as you suggest.